tiled/adapters/awkward_directory_container.py

```diff
@@ -0,0 +1,81 @@
+"""
+A directory containing awkward buffers, one file per form key.
+"""
+import collections.abc
+from pathlib import Path
+from typing import Any, Iterator
+
+
+class DirectoryContainer(collections.abc.MutableMapping[str, bytes]):
```

Comment:
I have a question about that. Is the `container` in the `awkward.py` is a `DirectoryContainer` ? I was not sure about that and I deduced it only from the name. If the `container` parameter in the `awkward.py` is a `DirectoryContainer` then there is a circular import since the `awkward_buffers.py` imports from `awkward.py`. In order to solve this problem, I put it inside another file.

Reply:
Update: The function `from_directory` in `awkwards_buffers.py` sends a `DirectoryContainer` to the `AwkwardAdapter` in `awkward.py` so I believe the `container` argument in the `AwkwardAdapter` is a `DirectoryContainer` so there is a circular import.